libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.html

```diff
@@ -3,21 +3,30 @@
   aria-haspopup="listbox"
   class="sky-autocomplete"
   role="combobox"
-  [attr.aria-expanded]="isDropdownVisible | async"
+  [attr.aria-expanded]="
+    isOpen && (showActionsArea || (isResultsVisible | async))
```

Comment:
This was not properly accounting for the dropdown being open just because of the actions area.